countersheet.py

```diff
@@ -1176,6 +1186,23 @@ def addregistrationmarks(self, xregistrationmarks, yregistrationmarks,
                                       docwidth - x1, y1,
                                       docwidth - x2, y2)
 
+    def find_foldingline_style(self):
+        return self.find_style("cs_foldstyle", DEFAULT_FOLDING_LINE_STYLE)
+
+    def create_foldingline(self, docwidth, dochight):
+        margin = max(self.registrationmarkslen,
```

Comment:
I think this calculation should also have the registration distance added after that other pull request has been merged?

Reply:
Yeah, of course you are right.